.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: ruby
 rvm:
-  - 2.1
```

Comment:
Would you be able to put this set of changes into a separate branch and make a new PR referencing #135?  You can keep the change here too for the time being for the sake of travis.